eviatar/README_Task1.txt

```diff
@@ -0,0 +1,7 @@
+Run the following commands for Deployment:
```

Comment:
Who is going to run these?
Do I need a human operator?

Reply:
Ideally, these commands should be automated within a CI/CD pipeline, reducing the need for a human operator. Tools like Jenkins, GitLab CI, or GitHub Actions can be configured to run these Terraform commands automatically when code is pushed to the repository. If manual execution is necessary, you should ensure that the operators are trained and aware of what each command does.